easybuild/easyconfigs/a/ARGoS/ARGoS-3.0.0-beta53-foss-2018b-Lua-5.2.4.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'ARGoS'
+version = '3.0.0-beta53'
+luaver = '5.2.4'
+versionsuffix = '-Lua-%s' % luaver
+
+homepage = 'http://www.argos-sim.info'
+description = """A parallel, multi-engine simulator for heterogeneous swarm robotics"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/ilpincy/argos3/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['849ebb1af31d01519a1f3d9731064eaa8041b2ed140bb699096912de1272214c']
+
+dependencies = [('CMake', '3.12.1')]
```

Comment:
@wpoely86 This should be `builddependencies`.

It also explains why @verdurin was hitting the `cmake: command not found` issue, I'm seeing the same problem...

Reply:
fixed in #8104

Reply:
OK, but why would that make it fail to find `cmake`?

Reply:
@wpoely86 Because you're re-defining `dependencies` below (and because @verdurin and I don't have `cmake` installed system-wide, while you do apparently).